provide-storage/backup-storage-provider/run-a-bsp-node.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Run a BSP Node
+description: Guide on how to set up, configure, and run a Backup Storage Provider (BSP) node on the DataHaven network using Docker.
+---
+
+# Run a BSP Node
+
+This guide explains how to prepare your project workspace, run the node with Docker, select the correct chain spec, and inject your BSP service key. By the end, you’ll have a fully configured BSP that is ready to join the DataHaven network.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- macOS or Linux operating system
+- Storage Capacity (minimum 1 TB, recommended 2 TB+)
+- [Docker](https://www.docker.com/){target=\_blank} and [Docker Compose](https://docs.docker.com/compose/install/){target=\_blank} installed and running
+- A BCSV key of scheme ECDSA (a 32 byte private key) for the BSP node's on-chain identity and signing
+- Sufficient account balance for deposits and collateral
+- Stable network connection
+- Open network ports (30333, optionally 9944)
+
+### Hardware Requirements
+
+BSPs have similar hardware requirements to MSPs as they store backup data and must reliably submit proofs of storage.
+
+#### Specifications
+
+| Component | Requirement |
+|-----------|-------------|
+| **CPU** | 8 physical cores @ 3.4 GHz (Intel Ice Lake+ or AMD Zen3+) |
+| **RAM** | 32 GB DDR4 ECC |
+| **Storage (System)** | 500 GB NVMe SSD (chain data) |
+| **Storage (User Data)** | 1 TB NVMe SSD or HDD (minimum) |
+| **Network** | 500 Mbit/s symmetric |
+
+#### Important Considerations
+
+- **Separate storage volumes**: Keep chain data and user data on separate volumes for better I/O performance
+- **Storage expansion**: Plan for growth; user data storage should be easily expandable
+- **max-storage-capacity**: Set this CLI flag to **80% of available physical disk space** to leave headroom for filesystem overhead and temporary files
+- **Cloud compatible**: BSPs can run effectively on cloud VPS with dedicated storage volumes
+- **Proof submission**: Ensure reliable network connectivity for timely proof submissions
+
+### Deposit Requirements
+
+- **Base Deposit**: 100 MOCK (`SpMinDeposit`)
+- **Per GiB**: 2 MOCK (`DepositPerData`)
+- **Formula**: 100 + (`capacity_in_gib` × 2) + buffer
```

Comment:
This is kind of confusing cause of the format that's being used. Like the stuff in parenthesis for top 2 items is just like this is what it's related to then in the last one it's like oh wait this is not what it's related to it's required for the formula